CONTRIBUTING.md

```diff
@@ -4,65 +4,18 @@
 
-- Generally, before developing enhancements to this charm, you should consider opening an issue explaining your use case.
+Please note that all the logic of this operator is stored in a shared library hosted [here](https://github.com/canonical/mongo-single-kernel-library).
+
+- Generally, before developing enhancements to this charm, you should consider opening an issue [on Single Kernel repository](https://github.com/canonical/mongo-single-kernel-library/issues) explaining your use case.
 - If you would like to chat with us about your use-cases or proposed implementation, you can reach us at [Canonical Mattermost public channel](https://chat.charmhub.io/charmhub/channels/charm-dev) or [Discourse](https://discourse.charmhub.io/).
 - Familiarising yourself with the [Charmed Operator Framework](https://juju.is/docs/sdk) library will help you a lot when working on new features or bug fixes.
 - All enhancements require review before being merged. Additionally, new code must pass the tests. Code review typically examines
-    - code quality
-    - test coverage
-    - user experience for Juju administrators of this charm.
+  - code quality
+  - test coverage
+  - user experience for Juju administrators of this charm.
 - Please help us out in ensuring easy to review branches by rebasing your pull request branch onto the `main` branch. This also avoids merge commits and creates a linear Git commit history.
-
-
-## Developing
-
-
-### Environment set up
-
-This operator charm can be deployed locally using [Juju on a localhost LXD cloud](https://juju.is/docs/olm/lxd). If you do not already have a Juju controller bootstrapped, you can set one up by doing the following:
-
-```
-# install requirements 
-sudo snap install charmcraft --classic
-sudo snap install lxd
-sudo snap install juju --classic
-
-# configure lxd
-sudo adduser $USER lxd
-newgrp lxd
-lxd init --auto
-lxc network set lxdbr0 ipv6.address none
-
-# bootstrap controller to lxd
-juju clouds
-juju bootstrap localhost overlord
-
-# Install environment dependencies
-sudo apt-get install python3-pip python3-venv -y --no-install-recommends
-python3 -m pip install pipx
-python3 -m pipx ensurepath
-          
-pipx install tox
-pipx install poetry
-pipx inject poetry poetry-plugin-export
-# TODO: Remove after https://github.com/python-poetry/poetry/pull/5980 is closed
-poetry config warnings.export false
-
-pipx install charmcraftcache
-```
-
-Clone this repository:
-```shell
-git clone https://github.com/canonical/mongodb-operator.git
-cd mongodb-operator/
-```
-
-Create and activate a virtualenv, and install the development requirements:
-```shell
-virtualenv -p python3 venv
-source venv/bin/activate
-```
-
+- Once the code has been merged on the Single Kernel repository, wait for a new version of the python package to be published, and create a PR on this repository that bumps the version of the package.
+- If you added some new interfaces, please don't forget to add them here.
```

Comment:
I would also before that they should open bump dependency PRs in all 4 repos. We shouldn't allow bumping in a single repo.